libs/geospatial/src/main/java/org/codice/ddf/libs/location/FIPSFormatConverter.java

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p/>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p/>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ **/
+package org.codice.ddf.libs.location;
+
+import java.util.Map;
+import java.util.MissingResourceException;
+import java.util.Objects;
+
+import org.apache.commons.lang.StringUtils;
+
+import ddf.security.PropertiesLoader;
+
+/**
+ * Converts a country code from FIPS 10-4 format to ISO 3166-1 alpha 3 format and vice versa.
+ */
+public class FIPSFormatConverter {
+
+    private final static String FIPS_10_4_to_ISO3 = "FIPS10-4toISO3.properties";
+
+    private static Map<String, String> countryCodesMap;
+
+    /**
+     * @param fipsCountryCode a FIPS 10-4 country code
+     * @return an ISO 3166-1 alpha-3 country code
+     */
+    public static String convertFipsToIso3(String fipsCountryCode) {
+        Map<String, String> countryCodesMap = getCountryCodesMap();
+
+        String iso3CountryCode = countryCodesMap.get(fipsCountryCode);
+        if (StringUtils.isBlank(iso3CountryCode)) {
+            throw new MissingResourceException(
+                    "Could not find a corresponding ISO 3166-1 alpha 3 country code given the FIPS"
+                            + " 10-4 country code: " + fipsCountryCode,
+                    FIPSFormatConverter.class.toString(),
+                    fipsCountryCode);
+        }
+        return iso3CountryCode;
+    }
+
+    /**
+     * @param iso3alphaCountryCode a FIPS 10-4 country code
+     * @return an ISO 3166-1 country code
+     */
+    public static String convertIso3ToFips(String iso3alphaCountryCode) {
+        Map<String, String> countryCodesMap = getCountryCodesMap();
+
+        String fipsCountryCode = countryCodesMap.entrySet()
```

Comment:
I originally tried that, but found out there are two FIPS countries that map to the same ISO alpha 3 country.